src/OrchardCore.Cms.Web/.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
@sebastienros or @jtkech should likely take a look to ensure it's something that should be included in the repo 😃